portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ACAdmin.testcase

```diff
@@ -89,4 +92,265 @@ definition {
 		}
 	}
 
+	@description = "Bug: LRAC-11339 | Automation ID: LRAC-11364 | Test Summary: A new user created in the admin page can not access that page but the user Test Test can"
+	@priority = "5"
+	test NewUserCanNotAccessAdminPage {
+		var userFirstName = "ac";
+		var userLastName = "ac";
+		var userScreenName = "ac";
+		var userEmailAddress = "ac@liferay.com";
+
+		task ("Get the URL link") {
+			var currentURL = Navigator.getCurrentURL();
+		}
+
+		task ("Navigate to the AC Admin page") {
+			Navigator.openSpecificURL(url = "${currentURL}admin");
+		}
+
+		task ("Go to control panel > Users > Users and Organizations") {
+			ProductMenuHelper.expandPanel(panel = "control panel");
+
+			ProductMenuHelper.expandCategory(category = "Users");
+
+			ProductMenuHelper.gotoPortlet(portlet = "Users and Organizations");
+		}
+
+		task ("Create a new user") {
+			LexiconEntry.gotoAdd();
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#DETAILS_SCREEN_NAME_FIELD",
+				value1 = "${userScreenName}");
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#DETAILS_EMAIL_ADDRESS_FIELD",
+				value1 = "${userEmailAddress}");
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#DETAILS_FIRST_NAME_FIELD",
+				value1 = "${userFirstName}");
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#DETAILS_LAST_NAME_FIELD",
+				value1 = "${userLastName}");
+
+			AssertClick(
+				locator1 = "Button#SUBMIT",
+				value1 = "Save");
+
+			UserNavigator.gotoUser(userScreenName = "${userScreenName}");
+
+			UserNavigator.gotoPassword();
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#PASSWORD_NEW_PASSWORD_FIELD",
+				value1 = "test");
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#PASSWORD_ENTER_AGAIN_FIELD",
+				value1 = "test");
+
+			Button.clickSave();
```

Comment:
Hey @manuferreira, couldn't the macros we use to put passwords on users in DXP work here? As this part of /admin is a DXP 7.2, I think you could try it, I'll leave two suggestions for you and I think if any works, maybe there's a way to create the users with the DXP macros as well:

- User.editPasswordCP
- JSONUser.setFirstPassword

Reply:
Hey Vini, I tested using a few macros and all of them failed at some point:
- User.editPassword
- User.editPasswordCP
- JSONUser.setFirstPassword
I also tried the JSON macro to create a user, but the user is only created on DXP and not where I want.